web/business-registry-dashboard/app/components/Table/AffiliatedEntity/index.vue

```diff
@@ -420,6 +413,7 @@ const mapDetailsWithEffectiveDate = (details: any[], row: any) => {
           <USelectMenu
             v-model="affStore.affiliations.pagination.limit"
             :options="affStore.paginationLimitOptions"
+            color="white"
```

Comment:
Partly fixes this:

Dev (incorrect)
<img width="216" height="117" alt="image" src="https://github.com/user-attachments/assets/cb6b8c36-f54f-47ea-bfc7-e55a6f15900f" />

Test (correct)
<img width="228" height="102" alt="image" src="https://github.com/user-attachments/assets/cbc0cea4-46fa-45de-9975-8cd2eb2c729d" />